docs/manifest.json

```diff
@@ -1127,6 +1127,12 @@
 		"markdown_source": "https://raw.githubusercontent.com/WordPress/gutenberg/master/packages/components/src/popover/README.md",
 		"parent": "components"
 	},
+	{
+		"title": "HorizontalRule",
```

Comment:
Automation works :)

Reply:
Kind of 😁 
I think this was generated by the recommit hook, but not included in the commit that added the README. I only noticed it because I did a second commit with the Usage section